scalableminds/webknossos
# Change parameter precedence when creating task from base

When creating a task from a base annotation, the starting position/rotation and bounding box as specified during task creation are now used and overwrite the ones from the original base annotation.

### URL of deployed dev instance (used for testing):
- https://taskbaseparameters.webknossos.xyz

### Steps to test:
- Create an explorative annotation, trace some, copy id
- Create a task, select base annotation, insert above id
- When requesting the task, you should start at the edit position + rotation as specified during task creation, not the ones from said explorative annotation.

### Issues:
- fixes #6200 
- contributes to #6241

------
